index.html

```diff
@@ -5,8 +5,199 @@
+                         Get your ticket, let us exchange knowledge, 
+                         best practices, submit feature requests and innovate.    
+                    </p>
+                </div>
+                <div class="headline-date">
+                    <h2>THU,Jun 30, 9:00 AM (EAT)</h2>
+                    <h5>@ Africa's Talking HQ
+                        Mzima Springs Rd, Lavington, Nairobi
+                        Nairobi, 00100</h5>
+                </div>
+            </div>
+        </section>
+        <section class="main-program-section">
+            <header class="program-header">
+                <h3>Agenda</h3>
+                <div class="divider"></div>
+            </header>
+            
+            <div class="articles">
+                <article class="program-article">
+
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/functional-training.png"/>                    </div>
+                    <div class="p-name">
+                        <h3>Warm-up</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Introduction and Pitching, 
+                            Form teams where need be. 
+                            Building/hacking, 
+                            Feedback, 
+                            Feature requests.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/laptop-coding.png"/>                    </div>
+                    <div class="p-name">
+                        <h3> Code</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Code in GitHub/GitLab/BitBucket and deploy in Docker/Automated
+                             CI/Travis CI/Bamboo <br>
+                        </p>
+    
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/widescreen-tv.png"/>
+                    </div>
+                    <div class="p-name">
+                        <h3>Demo</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Teams or individuals will demo what they have hacked for feedback or adoption.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/medal2--v1.png"/>                    </div>
```

Comment:
- [ OPTIONAL]  consider your website accessibility, by adding `alt ` value to your images so that it can increase your website accessibility to make your webpage more professional.